src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx

```diff
@@ -0,0 +1,722 @@
+            Cordova
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React Native
+        </TabsBarItem>
+        <TabsBarItem id="xamarin">
+            Xamarin
+        </TabsBarItem>
+        <TabsBarItem id="maui">
+            .NET MAUI
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java] 
+
+```java
+NewRelic.recordBreadcrumb(string $Name, map<string, object> $eventAttributes)
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+NewRelic.recordBreadcrumb(
+            breadcrumbName: String?,
+            attributes: MutableMap<String?, Any?>?
+        )
+```
+
+## Description [#description]
+
+This call creates and records a `MobileBreadcrumb` event, which can be queried with NRQL and in the [crash event trail](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/mobile-crash-event-trail). Mobile breadcrumbs are useful for crash analysis; they should be created for app activity that may be helpful for troubleshooting crashes.
+
+In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-attribute), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
+
+For related Android SDK API calls, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/guide-custom-instrumentation-api#send-insights).
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width="30%">
+        Parameter
+      </th>
+
+      <th width={200}>
+        Type
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `Name`
+      </td>
+
+      <td>
+        `string`
+      </td> 
+
+      <td>
+        Required. The name you want to give to the breadcrumb event.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        `eventAttributes`
+      </td>
+
+      <td>
+        `dictionary`
+      </td>
+      <td>
+        Optional. A map that includes a list of attributes of the breadcrumb event. Create attributes for any event descriptors you think will be useful.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Return values [#return-values]
+
+Returns `true` if the event is recorded successfully, or `false` if not.
+
+## Examples [#examples]
+
+Here's an example of a simple breadcrumb event call:
+
+### Java [#java]
+
+```java
+Map attributes = new HashMap();
+attributes.put("attributeName1", "value1");
+attributes.put("attributeName1", 2);
+boolean eventRecorded = NewRelic.recordBreadcrumb("Name", attributes);
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
val map = mutableMapOf<String,Any>("attributeName1" to "value1", "attributeName1" to 2)
        val eventRecorded = NewRelic.recordBreadcrumb("Name", map)

Reply:
```suggestion
```kotlin

val map = mutableMapOf<String,Any>("attributeName1" to "value1", "attributeName1" to 2)
val eventRecorded = NewRelic.recordBreadcrumb("Name", map)

```
```